mmv1/products/cloudids/terraform.yaml

```diff
@@ -28,5 +28,5 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       location: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
       name: !ruby/object:Overrides::Terraform::PropertyOverride
-        custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+        custom_flatten: 'templates/terraform/custom_flatten/id_from_name.erb'
```

Comment:
question: is this change needed? what issue will it run into if we use the previous flattener?

Reply:
I remember I did this change because I had some error doing an "update" for the "threat_exception" field, in the end the root cause was something else. 
Therefore I don't think this change matters much for now. 

The API team is still developing new features on this particular resource. I suggest leaving the change as it is since it  looks to work for create/update test. If this breaks in the future we can always take further action.